### DISCUSSION TITLE
Example SaaS Signup Administration Pages

### DISCUSSION THREAD

**Discussion by everett564** [Azure/azure-saas#216] [General]
Hi, 

I am looking to explore using this template for my SaaS application development, however I am having difficulty launching this template locally. The only thing I need at the moment are some example screenshots of this application specifically ones for the [Saas.SignupAdministration](https://github.com/Azure/azure-saas/tree/main/src/Saas.SignupAdministration) solution. Can you please post some example screenshots of the signup process here?

Thank you!

**Comment by 1iveowl** [comment]
Hi @everett564,

Could you elaborate on what you have done to launch the ASDK locally? The ASDK is not a template, rather to run it please consult the [Quick Start Guide](https://azure.github.io/azure-saas/quick-start/).

The sign-up page looks like this.
![image](https://user-images.githubusercontent.com/12871120/235351665-a666aafa-8d2d-4db6-826c-7b461e9bbf1f.png).

**Reply by everett564** [reply]
Hi @1iveowl, 

Yes, I was going through that doc, but got stuck on a few steps. Thank you for this screenshot, this is some of what I am looking for. Can you also send screenshots of the Organizational registration/ any other pages you are willing to share? Thanks!
